layers/transportation/update_route_member.sql

```diff
@@ -84,14 +89,20 @@ CREATE INDEX IF NOT EXISTS osm_route_member_network_type_idx ON osm_route_member
 CREATE INDEX IF NOT EXISTS osm_highway_linestring_osm_id_idx ON osm_highway_linestring ("osm_id");
 CREATE INDEX IF NOT EXISTS osm_highway_linestring_gen_z11_osm_id_idx ON osm_highway_linestring_gen_z11 ("osm_id");
 
-ALTER TABLE osm_route_member ADD COLUMN IF NOT EXISTS concurrency_index int;
+ALTER TABLE osm_route_member ADD COLUMN IF NOT EXISTS concurrency_index int,
+                             ADD COLUMN IF NOT EXISTS rank int;
 
 -- One-time load of concurrency indexes; updates occur via trigger
-INSERT INTO osm_route_member (id, osm_id, concurrency_index)
+INSERT INTO osm_route_member (id, osm_id, concurrency_index, rank)
   SELECT
     id,
     osm_id,
-    DENSE_RANK() over (PARTITION BY member ORDER BY network_type, network, LENGTH(ref), ref) AS concurrency_index
+    DENSE_RANK() over (PARTITION BY member ORDER BY network_type, network, LENGTH(ref), ref) AS concurrency_index,
+    CASE
+         WHEN network IN ('iwn', 'nwn', 'rwn') THEN 1
+         WHEN network = 'lwn' THEN 2
+         WHEN COALESCE(osmc_symbol, colour) <> '' THEN 2
+    END AS rank
   FROM osm_route_member
   ON CONFLICT (id, osm_id) DO UPDATE SET concurrency_index = EXCLUDED.concurrency_index;
```

Comment:
I was unable to add `rank` values for hiking routes. This fixes it.

```suggestion
  ON CONFLICT (id, osm_id) DO UPDATE SET concurrency_index = EXCLUDED.concurrency_index, rank = EXCLUDED.rank;
```